lib/spack/spack/build_systems/intel.py

```diff
@@ -103,6 +136,742 @@ def _filtered_components(self):
 
         return matches
 
+    @property
+    def _want_shared(self):
+        return ('+shared' in self.spec)
```

Comment:
Not sure if this really adds anything...

Reply:
I placed this in a function to encapsulate the hard-coded variant string and its testing.

Also, the function name improves the signal-to-noise ratio for ``*`` (whole-keyword search) in vim 😁, compared to ``shared``.

Reply:
> encapsulate the hard-coded variant string and its testing.

‘+shared’ is convention in Spack, I also don’t think you win much by doing this

Reply:
Hmm. My parseltongue is marbled by a perl.